Depricate JSON FileStore (#196)
netbirdio/docs: src/pages/selfhosted/sqlite-store.mdx

```

### Rollback to JSON file store
<Note>

> Starting from version [0.28.0](https://github.com/netbirdio/netbird/releases/tag/v0.28.0), support for JSON file storage is deprecated.
> The option to roll back from SQLite to JSON will not be supported, so users will need to use SQLite as default store.

Reviewer: ```suggestion
Since version [0.28.0](https://github.com/netbirdio/netbird/releases/tag/v0.28.0), support for JSON file storage has been removed, and the option to roll back to JSON store is no longer supported.
```
